packages/editor/src/editor-styles.scss

```diff
@@ -42,17 +42,6 @@ h6 {
 	font-size: 0.8em;
 }
 
-// Adjust line spacing for larger headings.
```

Comment:
:wave: @jasmussen Want to double-check why these styles were useful for? Is it the case that this are here to prevent wp-admin styles from overriding editor styles?

If so, we may be able to get rid of some of them: one of the [changes](https://github.com/WordPress/gutenberg/pull/21215/files#diff-f2f5efb13e0e16e61312aa9b0c028ae1) introduced in the line-height control PR recently was to add a line-height property by default to the headings block. Same for color below in [this PR](https://github.com/WordPress/gutenberg/pull/21215/files#diff-f2f5efb13e0e16e61312aa9b0c028ae1).

Reply:
Thanks for the ping. Please get rid of as much of this as you like, so as to make sense of the global style system. While I can't 100% recall exactly why I added those spacings in the first place, a very good bet is that it had to do with inherited CSS bleed from wp-admin.